src/test/java/com/amazon/ion/conformance/structure.kt

```diff
@@ -162,8 +162,8 @@ private fun ParserState.readExtension(): List<DynamicNode> {
 private fun ParserState.readContinuation(): DynamicNode {
     val continuation = sexp.tailFrom(pos)
 
-    val firstExpression = continuation.first()
-    firstExpression as? SeqElement ?: builder.reportSyntaxError(firstExpression, "continuation")
+    val firstExpression = continuation.firstOrNull()
+    firstExpression as? SeqElement ?: builder.reportSyntaxError(sexp, "continuation")
```

Comment:
🗺️ If there was no `firstExpression`, calling `first()` would throw `NoSuchElementException`, which didn't have any information about where the syntax error occurred in the conformance DSL. This change just ensures that more syntax errors are reported in a useful way.